transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -458,22 +468,22 @@ public final ChannelHandlerInvoker invoker() {
         }
 
         @Override
-        public final ChannelOutboundBuffer outboundBuffer() {
+        public ChannelOutboundBuffer outboundBuffer() {
             return outboundBuffer;
```

Comment:
`ChannelOutboundBuffer` is a final class. What would you do with this method if it's not final?